### DISCUSSION TITLE
New locus: TAF1 X-linked dystonia-parkinsonism (XDP)

### DISCUSSION THREAD

**Discussion by hdashnow** [dashnowlab/STRchive#17] [Locus updates and new loci]
See old PR #4 

This locus may need more specific boundaries before we can include it. Will need a fresh PR since the database format has changed since then.

**Comment by laurelhiatt** [comment]
chromosome X
71453055–
71453129 from PMID: 38876750

**Comment by hdashnow** [comment]
I'm putting some specific from the paper here to help update the json:

From PMID: 38876750

> A pathogenic intronic 2·6 kb fragment, comprising a short interspersed nuclear element, a variable number of tandem repeats, and an Alu element, is inserted in intron 32 of TAF1 in individuals with X-linked dystonia parkinsonism. 787980 The variable number of tandem repeats is a hexameric AGAGGG repeat expansion of 35–52 repeats, the length of which is polymorphic among patients with X-linked dystonia parkinsonism and inversely correlates with age of onset. 78798081 The AGAGGG expansion appears to be the primary cause of the disease, although the participation of other elements of the intronic fragment cannot be excluded.
> ...
> In transcriptomic analyses done in human cells in vitro, three defects in TAF1 expression have been described: aberrant RNA splicing, increased partial retention of intron 32, and decreased transcription of 3′ exons that reduces the full-length transcript. All of these transcriptional defects were rescued by CRISPR-based excision of the intronic fragment. 80 Therefore, a partial loss-of-function of TAF1 due to the intronic repeat expansion itself has been hypothesised as the primary mechanism underlying neurodegeneration in X-linked dystonia parkinsonism. 7880 Nonetheless, additional mechanisms could be at play since missense variants in TAF1 are known to be associated with intellectual disability, 87 but not parkinsonism or dystonia, and the complete loss of TAF1 is lethal.


  | Gene; genomic location (GRCh38 or hg38) | Location in gene | Reference motif * | Reference size, number of repeat units | Pathogenic repeat * | Pathogenic repeat size, number of repeat units | Pathogenic mechanism | Ethnic and geographical distribution | Main clinical features
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
X-linked dystonia parkinsonism 7980 | TAF1 ; chromosome X 71453055–71453129 | Intron (retrotransposon) | AGAGGG (AGAGGG) | 4 | AGAGGG (AGAGGG) | 35–52 | Altered splicing with intron retention, haploinsufficiency | Filipino | Focal and generalised dystonia, parkinsonism, cognitive dysfunction



Primary source papers:

Makino S, Kaji R, Ando S, et. al.: Reduced neuron-specific expression of the TAF1 gene is associated with X-linked dystonia-parkinsonism. Am J Hum Genet 2007; 80: pp. 393-406.
http://dx.doi.org/10.1086/512129

Bragg DC, Mangkalaphiban K, Vaine CA, et. al.: Disease onset in X-linked dystonia-parkinsonism correlates with expansion of a hexameric repeat within an SVA retrotransposon in TAF1 . Proc Natl Acad Sci USA 2017; 114: pp. E11020-E11028.

Aneichyk T, Hendriks WT, Yadav R, et. al.: Dissecting the causal mechanism of X-linked dystonia-parkinsonism by integrating genome and transcriptome assembly. Cell 2018; 172: pp. 897-909.
http://dx.doi.org/10.1016/j.cell.2018.02.011

**Comment by hdashnow** [comment]
New PR for this: #43
I'd love feedback

**Comment by hdashnow** [comment]
Should the year be the date the disease was described and associated with the gene (2007) or the date that the likely causal variant was identified as a tandem repeat (2017)?

**Comment by laurelhiatt** [comment]
Tandem repeat, in my opinion Best,LaurelOn Oct 23, 2024, at 9:47 PM, Harriet Dashnow ***@***.***> wrote:﻿
Should the year be the date the disease was described and associated with the gene (2007) or the date that the likely causal variant was identified as a tandem repeat (2017)?

**Comment by hdashnow** [comment]
This locus was added with #43. Closing discussion.
